test/integration/specs/narrative_basic_test.js

```diff
@@ -16,7 +16,7 @@ describe('Narrative tree page with login', () => {
     });
 
     it('opens a narrative', async () => {
-        await browser.url(makeURL(`narrative/${testCases.CASE_1.narrativeId}`));
```

Comment:
`makeURL` (which just prepends the BASE_URL) isn't necessary. wdio does that for us. Was fiddling around here when I found that - will update other specs in a later PR.